### DISCUSSION TITLE
PyCharm does not show folder structure of repo generated from theorist template

### DISCUSSION THREAD

**Discussion by musslick** [AutoResearch/autora#363] [Core Contributions] [answered]
I created a template from autora-theorist: https://github.com/AutoResearch/autora-experimentalist-template

After cloning the new repository and opening it with PyCharm, PyCharm does not seem to show me the folder structure. I can see them in the finder but Pycharm doesn't show any of the src folders.
<img width="348" alt="image" src="https://user-images.githubusercontent.com/10374030/234806122-87d98b3b-14aa-4725-8dc0-4286f2face72.png">
@hollandjg @younesStrittmatter Any idea what is going wrong?

**Comment by younesStrittmatter** [comment, marked as answer]
@hollandjg I think this has to do with the .idea or .vscode folder in the template. after deleting and cloning the folders show up on my side

**Reply by hollandjg** [reply]
We could take out the .idea and .vscode folders completely. @younesStrittmatter, what do you think?

**Reply by hollandjg** [reply]
We could include a guide to setting up your environment with VSCode, PyCharm somewhere, so that people have the full experience too.

**Reply by musslick** [reply]
I think that would be great!

**Comment by younesStrittmatter** [comment]
that makes sense to me. I'll take them out of the template

**Comment by younesStrittmatter** [comment]
I deleted the folders. Hope it works now

**Reply by musslick** [reply]
This helped, thank you!
